worker/worker.py

```diff
@@ -1303,12 +1304,18 @@ def fetch_and_handle_task(worker_info, password, remote, lock_file, current_stat
         return False
 
     try:
```

Comment:
I wonder if we can shrink the code in this `try`.
If I'm not wrong `get_remaining_github_api_calls()` already deals with its exception.

The `try` here is dealing only with the `send_api_post_request()` exceptions, one directly in `fetch_and_handle_task()` and one nested in `verify_worker_version()`

Reply:
Ok I changed it. Have a look.

Reply:
Reading and testing.